MAUI/Maps/zoom-pan.md

```diff
@@ -0,0 +1,633 @@
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfMaps maps = new SfMaps();
+MapShapeLayer layer = new MapShapeLayer();
+layer.ShapesSource = MapSource.FromUri(new Uri("https://cdn.syncfusion.com/maps/map-data/world-map.json"));
+MapZoomPanBehavior zoomPanBehavior = new MapZoomPanBehavior();
+zoomPanBehavior.ZoomLevel = 1;
+layer.ZoomPanBehavior = zoomPanBehavior;
+maps.Layer = layer;
+this.Content = maps;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+**Tile layer**
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <map:SfMaps>
+    <map:SfMaps.Layer>
+        <map:MapTileLayer UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png">
+              <map:MapTileLayer.ZoomPanBehavior>
+                 <map:MapZoomPanBehavior ZoomLevel = "1" />
+              </map:MapTileLayer.ZoomPanBehavior>
+        </map:MapTileLayer>
+    </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfMaps map = new SfMaps();
+MapTileLayer tileLayer = new MapTileLayer();
+tileLayer.UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png";
+MapZoomPanBehavior zoomPanBehavior = new MapZoomPanBehavior();
+zoomPanBehavior.ZoomLevel = 1;
+tileLayer.ZoomPanBehavior = zoomPanBehavior;
+map.Layer = tileLayer;
+this.Content = map;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Osm map default](images/zoom-pan/osm_map.png)
+
+## Customizing the zoom level
+
+You can set the zoom level of the map layer by using `MapZoomPanBehavior.ZoomLevel` property.
+
+The default `MapZoomPanBehavior.ZoomLevel` value is 1 which will show the whole map in the viewport for `MapTileLayer`.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <map:SfMaps>
+   <map:SfMaps.Layer>
+        <map:MapTileLayer UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png" >
+              <map:MapTileLayer.Center>
+                  <map:MapLatLng Latitude = "27.1751"
+                                 Longitude="78.0421">
+                  </map:MapLatLng>
+              </map:MapTileLayer.Center>
+              <map:MapTileLayer.ZoomPanBehavior>
+                  <map:MapZoomPanBehavior ZoomLevel = "5" />
+              </map:MapTileLayer.ZoomPanBehavior>
+        </map:MapTileLayer>
+   </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfMaps map = new SfMaps();
+MapTileLayer tileLayer = new MapTileLayer();
+tileLayer.UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png";
+tileLayer.Center = new MapLatLng(27.1751, 78.0421);
+MapZoomPanBehavior zoomPanBehavior = new MapZoomPanBehavior();
+zoomPanBehavior.ZoomLevel = 5;
+tileLayer.ZoomPanBehavior = zoomPanBehavior;
+map.Layer = tileLayer;
+this.Content = map;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![OSM map zoomlevel](images/zoom-pan/osm_maps_zoomlevel.png)
+
+## Enable zooming 
+
+The zooming feature enables you zoom in and zoom out the maps to show the in-depth information. To enable zooming, set the `EnableZooming` property to `true`. The default value of the `EnableZooming` is `true`.
```

Comment:
The zooming feature enables you to zoom in and zoom out the maps to show in-depth information. To enable zooming, set the `EnableZooming` property to `true.` The default value of the `EnableZooming` is `true.`

Reply:
Changed the content